package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@doist/reactist",
     "description": "Open source React components by Doist",
     "author": "Henning Muszynski <henning@doist.com> (http://doist.com)",
-    "version": "7.2.1",
+    "version": "9.0.0",
```

Comment:
Why not `8.0.0`?

Reply:
The [v8.0.0](https://github.com/Doist/reactist/tree/v8.0.0) tag already exists upstream, and I wasn't sure if there'd be any issues if I force pushed a tag, or if it's even allowed from the releases page, so I figured it's easier to just skip a version.

(There's a note on this in the changelog document)